### PR TITLE
Publish to the dist/ folder to conform with other NPM artifacts

### PR DESCRIPTION
**Pull request by rchild-okta**
## Description:

Two fixes:

1. Moves **_site** to **dist** to conform with other NPM artifacts
2. Small fix to no longer load typescript or google analytics in localhost (was throwing errors in the phantomjs tests).

## Type of Pull Request:
- [x] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* [OKTA-125216](https://oktainc.atlassian.net/browse/OKTA-125216)

## How Has This Been Tested?
- [x] I have built this locally and verified that it does not break existing functionality.

## Primary Reviewers:
- *Tag reviewer(s)*
@lboyette-okta @sergiymelnyk-okta

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,22 +47,21 @@ and generate the final site content.
...
+2. Make changes in the **_source** directory
+3. Compile changes into the **dist** directory `bundle exec jekyll serve --watch`
 4. Navigate the site and validate your changes
 5. Stop Jekyll with `ctrl+c`
 6. Discard changed files (they have "localhost:4000" in link URLs instead of "developer.okta.com") with `git checkout .`
-7. Build again with `rm _site/*; bundle exec jekyll build`
-8. Sync the built _site files with the checked-in code with: `rsync -av _site/ ./`
-9. Git commit and push changes to github. When ready for review create a pull request and mention the users you want to
+7. Build a production version with `npm run build-prod`
+8. Commit and push changes to github. When ready for review create a pull request and mention the users you want to
```
Comment (lboyette-okta) [inline README.md]: nit: github => GitHub - we should standardize on casing in the doc
nit: When ready for review create => When ready for review, create

also, does it make sense to show sample git commands? We do show how to push later on, so maybe it's not necessary. `git commit -a -m "My changes!"` `git push origin YOUR_BRANCH`

File: _source/_includes/head.html [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
-      kitId: 'jff5neq',
-      scriptTimeout: 3000,
-      async: true
-    },
-    h=d.documentElement,t=setTimeout(function(){h.className=h.className.replace(/\bwf-loading\b/g,"")+" wf-inactive";},config.scriptTimeout),tk=d.createElement("script"),f=false,s=d.getElementsByTagName("script")[0],a;h.className+=" wf-loading";tk.src='https://use.typekit.net/'+config.kitId+'.js';tk.async=true;tk.onload=tk.onreadystatechange=function(){a=this.readyState;if(f||a&&a!="complete"&&a!="loaded")return;f=true;clearTimeout(t);try{Typekit.load(config)}catch(e){}};s.parentNode.insertBefore(tk,s)
-  })(document);
+    const isProduction = window.location.hostname === 'developer.okta.com';
+    if (isProduction) {
+      // TypeKit
+      (function(d) {
```
Comment (lboyette-okta) [inline _source/_includes/head.html]: Will this mess up the fonts if we run it locally?

File: README.md (hunk 1 above)
Comment (rchild-okta) [inline README.md]: I'll make these changes, but in general have stayed away from modifying this readme too heavily - there are a bunch of other things that can (and should) be fixed in a followup pr.

File: _source/_includes/head.html (hunk 2 above)
Comment (rchild-okta) [inline _source/_includes/head.html]: It won't - it'll just use the standard fallback font. Right now it doesn't show the correct font anyway - we haven't listed localhost as a valid domain in typekit.

**Pull request merged**